tools/tm-bench/main.go

```diff
@@ -93,6 +95,19 @@ Examples:
 		"broadcast_tx_"+broadcastTxMethod,
 	)
 
+	//catch Interrupt and quit tm-bench
+	go func() {
```

Comment:
This should be moved up, before the first side-effectful call on line 86. It also doesn't need to be wrapped in a goroutine entirely, only the part which ranges over the channel.